zokrates_stdlib/stdlib/hashes/pedersen/512bitBool.zok

```diff
@@ -1,8 +1,8 @@
 import "utils/multiplexer/lookup3bitSigned" as sel3s;
 import "utils/multiplexer/lookup2bit" as sel2;
-import "ecc/edwardsAdd" as add;
-import "ecc/edwardsCompress" as edwardsCompress;
-from "ecc/babyjubjubParams" import BABYJUBJUB_PARAMS;
+import "ecc/babyjubjub/compress";
+from "ecc/edwards" import add;
+from "ecc/babyjubjub/params" import EDWARDS_A, EDWARDS_D;
```

Comment:
The pedersen hash works currently only with `bn128` but from my understanding it should also work with bls12_381 ([see](https://github.com/iden3/iden3-docs/blob/master/source/iden3_repos/research/publications/zkproof-standards-workshop-2/pedersen-hash/pedersen.rst#background).

I would say, that implementing the pedersen hash for both curves is out of the scope of this ticket. Should I maybe create a follow-up ticket for the it? I would also add the `#pragma curve bn128` macro in the mean time for the pedersen hash, to avoid misunderstandings.

Reply:
Yes! Feel free to open a PR pointing to this branch